docker-compose.yml

```diff
@@ -306,6 +306,14 @@ services:
       - "logging=true"
     restart: always
     logging: *default-logging
+  kbo-data-sync:
+    image: kbo-data-sync-service:0.0.1
+    links:
+      - db:database
+    labels:
+      - "logging=true"
+    restart: always
+    logging: *default-logging
 
```

Comment:
**Remark:** Be sure to correct this configuration here once the service itself is updated such that the correct (version of the) service is used instead of two distinct ones.
As a side note, I would put temporary configurations used during development in the `docker-compose.dev.yml` file instead of here.